Library/Formula/mdbtools.rb

```diff
@@ -5,12 +5,16 @@ class Mdbtools < Formula
   sha256 "dcf310dc7b07e7ad2f9f6be16047dc81312cfe1ab1bd94d0fa739c8059af0b16"
 
   option "with-man-pages", "Build manual pages"
+  option "with-iodbc", "Build with iODBC"
+  option "with-unixodbc", "Build with unixODBC"
 
   depends_on "pkg-config" => :build
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "txt2man" => :build if build.with? "man-pages"
+  depends_on "libiodbc" => :optional if build.with? "iodbc"
```

Comment:
Just use `:optional` here and remove the `build.with?` and it'll generate an option called `--with-libiodbc`